schema/rdl_schema_0.1.json

```diff
@@ -949,6 +918,61 @@
 				"TCY"
 			]
 		},
+		"Resource": {
+			"title": "Resource",
+			"type": "object",
+			"description": "Information about an individual version of the dataset.",
+			"required": [
+				"id",
+				"title",
+				"description",
+				"media_type",
+				"url"
+			],
+			"properties": {
+				"id": {
+					"title": "Resource identifier",
+					"type": "string",
+					"description": "An identifier for this resource. This should be internally unique."
+				},
+				"title": {
+					"title": "Resource title",
+					"type": "string",
+					"description": "A title for this resource."
+				},
+				"description": {
+					"title": "Resource description",
+					"type": "string",
+					"description":"Additional description of the resource not already covered by th eother resource fields." 
+				},
+				"media_type": {
+					"title": "Media type",
+					"type": "string",
+					"description": "The media type of the resource taken from the [IANA Media Types codelist](https://www.iana.org/assignments/media-types/media-types.xhtml). For example a custom binary file has media_type 'application/octet-stream. A geojson file has the media_type 'application/geo+json'."	
```

Comment:
We can include the IANA codelist as a CSV file in order to support validation.

Reply:
Once https://github.com/GFDRR/rdl-standard/pull/110 is merged and `dev` is merged into this branch, I can add the CSV file and a script to keep it up to date.

Reply:
Done in https://github.com/GFDRR/rdl-standard/pull/102/commits/fad3756cdc9a8f22377adf58c690f4a9152e2ff8 and https://github.com/GFDRR/rdl-standard/pull/102/commits/e6d6e4005aec00711e4b9bb952f3489a6cf235cc